doc/articles/get-started-vsmac.md

```diff
@@ -62,4 +62,25 @@ Building for WebAssembly takes a few more steps:
 1. Once installed type `dotnet serve` (or `~/.dotnet/tools/dotnet-serve`).
 1. Navigate to the URL presented by the tool to run your application
 
+### Using Visual Studio Code snippets to add new XAML files
+
+There is no similar extension at the moment available for Visual Studio for Mac, so we invite you to use the [Uno Platform Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=unoplatform.vscode) Extension.
+This extension comes with [snippets](get-started-vscode.md#using-code-snippets) to easily add new files like `Page`, `UserControl`, and `ResourceDictionary` in addition to other additional useful snippets to enhance your coding experience on Mac for the `XAML` files.
+
+Visit [our getting started section with Visual Studio Code](get-started-vscode.md) for more details.
+
+You're all set! You can now head to [our tutorials](getting-started-tutorial-1.md) on how to work on your Uno Platform app.
+
+## Troubleshooting
+
+### SDK iOS version incompatible with Xcode version
+Sometimes the SDK iOS version is incompatible with the Xcode version. To workaround this issue you can open a terminal, and type the following command:
+
+```bash
+sudo dotnet workload update
+```
+
+### iOS Simulators do not appear
+The issue is related to Visual Studio for Mac that changes `net7.0-ios` to `net7.0-ios16.1` which causes Visual Studio to not be able to load the list of simulators. A workaround for this will be to roll back to `net7.0-ios`, and you will be able to see the simulators again.
```

Comment:
minor nitpicks

> that changes `net7.0-ios` to `net7.0-ios16.1`

It can be different, like `net7.0-ios16.4` or it could be `net8.0...`. Maybe something like:

> that changes a general TFM, like `net7.0-ios`, to a version specific TFM, like `net7.0-ios16.1`, which ...

> be to roll back to `net7.0-ios`

It's not clear that this change was made to the `.csproj` and how that file can be opened/edited inside VSfM.

Reply:
Comments resolved, let me know what you think @spouliot